data_steward/validation/main.py

```diff
@@ -696,8 +697,10 @@ def perform_validation_on_file(file_name, found_file_names, hpo_id,
 
 
 def _validation_done(bucket, folder):
-    if gcs_utils.get_metadata(bucket=bucket,
-                              name=folder + common.PROCESSED_TXT) is not None:
+    storage_client = StorageClient()
+    sc_bucket = storage_client.get_bucket(bucket)
+    bucket_blob = sc_bucket.blob(f'{folder}{common.PROCESSED_TXT}')
+    if storage_client.get_blob_metadata(bucket_blob):
         return True
     return False
```

Comment:
There is some redundancy here, since you are already checking if something is true or not and just passing the booleans. You could simply return the whole thing instead and achieve the same result. This existed in the old code, so I understand you were just following the existing convention. But I'd say look at the usages of _validation_done and see if it makes sense and replace it.
```suggestion
    return storage_client.get_blob_metadata(bucket_blob)
```